src/pages/InputDemo/InputDemo.jsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import TextField from '../../components/TextField/TextField';
+import SelectField from '../../components/SelectField/selectField';
+import RadioGroup from '../../components/RadioGroup/radioGroup';
+import { selectOptions, radioOptionsCricket, radioOptionsFootball } from '../../config/constant';
+
+
+class InputDemo extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      name: '',
+      sport: '',
+      cricket: '',
+      football: '',
+    };
+  }
+
+  onChangeTextField = (event) => {
+    this.setState({ name: event.target.value });
+  }
+
+  onChangeSelectOptions = (event) => {
+    let { cricket, football } = this.state;
+    cricket = '';
```

Comment:
just set the sport and position="".
Make sure value of sport is not set to select. It should be one of these '', 'cricket', 'football'